### ISSUE TITLE
@media queries should have 1px difference for min-width and max-width values

### ISSUE THREAD

**Issue by frol** [near/near-explorer#639]
## Story

> I observed in quite a few places that @media (min-width) and @media (max-width) had the same values. It should be something like max-width: 767px and min-width: 768px (one pixel difference), etc. In validators.jsx the media min and max values are the same.

## Acceptance Criteria

- [ ] The affected widths should be manually tested and confirmed that those do not overlap and apply two sets of rules at the same time
- [ ] Mobile ok

**Comment by shelegdmitriy** [comment]
@frol you are right. According to [doc](https://getbootstrap.com/docs/5.0/layout/breakpoints/) we should do like so

`
@media (min-width: 768px) and (max-width: 1199.98px) { ... }
`

And `.98` ending is needed to work fine on retina screens.
